src/policy/policy.h

```diff
@@ -20,7 +20,9 @@ class CFeeRate;
 class CScript;
 
 /** Default for -blockmaxweight, which controls the range of block weights the mining code will create **/
-static constexpr unsigned int DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT - 4000};
+static constexpr unsigned int DEFAULT_BLOCK_MAX_WEIGHT{MAX_BLOCK_WEIGHT};
+/** The default reserved weight for the fixed-size block header, transaction count and coinbase transaction. **/
```

Comment:
In c961c509bc5d4a8c9e9197cad001d2b7598c8bd7 "miner: bugfix: fix duplicate weight reservation in block assembler": I'd rather keep the explanation in `types.h` since `BlockCreateOptions` is part of an external interface that should make sense without having to dig into internals.

Reply:
Done in this [push](https://github.com/bitcoin/bitcoin/compare/e7df348288e81dae104df7f8c984cb8eca57da35..6709e185b93d97b9191597acfecb25c33ab43a0a)
Also updated the release note to fix grammar